usautobuild/actions/builder.py

```diff
@@ -174,14 +176,19 @@ def start_building(self) -> None:
 
         for target in self.config.target_platforms:
             log.debug(f"Starting build for {target}...")
+            build_time = time.time()
 
             try:
                 self.build(target)
             except BuildFailed:
                 if self.config.abort_on_build_fail:
-                    log.error(f"Build for {target} failed and config is set to abort on fail!")
+                    final_build_time = ':.2f'.format((time.time() - build_time))
+                    final_time = ':.2f'.format((time.time() - time_start))
+                    log.error(f"Build for {target} failed and config is set to abort on fail!\n Build Time Took: {final_build_time}.\n Total Time Took: {final_time}")
```

Comment:
is build time relevant when build failed?

Reply:
wdym?